configs/_base_/recog_models/asternet.py

```diff
@@ -0,0 +1,28 @@
+# model
+label_convertor = dict(
+    type='AttnConvertor', dict_type='DICT36', with_unknown=True, lower=True)
+
+model = dict(
+    type='ASTERNet',
+    preprocessor=dict(
+        type='TPSPreprocessor',
+        img_size=(32, 100),
+        rectified_img_size=(32, 100),
+        num_img_channel=3),
+    backbone=dict(
+        type='ResNetABI',
+        arch_settings=[3, 4, 6, 6, 3],
+        strides=[2, 2, [2, 1], [2, 1], [2, 1]],
+        init_cfg=dict(type='Xavier', layer='Conv2d')),
```

Comment:
We have recently refactored the [ResNet structure](https://github.com/open-mmlab/mmocr/pull/809). ResNet in ASTER can be implemented using this configuration
```python
backbone = dict(
        type='ResNet'
        in_channels=3,
        stem_channels=[64, 128],
        block_cfgs=dict(type='BasicBlock', use_conv1x1='True'),
        arch_layers=[3, 4, 6, 6, 3],
        arch_channels=[32, 64, 128, 256, 512],
        strides=[(2, 2), (2, 2), (2, 1), (2, 1), (2, 1)])
```